.travis.yml

```diff
@@ -2,19 +2,16 @@ language: python
 python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
+  - eggs
+  - test.plone_addon/eggs
```

Comment:
@jensens there seem to be issues with the caching of this directory and I think they are caused because the name is the same in both cases.